tokens/style-dictionary.js

```diff
@@ -65,165 +80,188 @@ const colorTransform = (token, theme) => {
  * 2. 'theme' to output only theme's variables (e.g, 'light' or 'dark'), if theme is not provided - only
  * core tokens are built.
  */
-const createCustomCSSVariables = ({
+const createCustomCSSVariables = async ({
   formatterArgs,
   themeVariant,
-}) => {
+}, StyleDictionary) => {
+  // eslint-disable-next-line import/no-unresolved
+  const { sortByReference, usesReferences, getReferences } = (await import('style-dictionary/utils'));
```

Comment:
I wonder if it'd make sense to abstract a helper util function to make these utils accessible, e.g. 

```jsx
/* eslint-disable import/no-unresolved */
const getStyleDictionaryUtils = async () => import('style-dictionary/utils');
/* eslint-enabled import/no-unresolved */

// createCustomCSSVariables
const { sortByReference, usesReferences, getReferences } = await getStyleDictionaryUtils();
```

`getStyleDictionaryUtils` could also be used within `initializeStyleDictionary`, e.g.:

```jsx
const initializeStyleDictionary = async () => {
  // eslint-disable-next-line import/no-unresolved
  const StyleDictionary = (await import('style-dictionary')).default;
  const { getReferences } = await getStyleDictionaryUtils();
 
  // ...
}
```

Reply:
[inform] Implemented in follow-up PR: https://github.com/openedx/paragon/pull/3203